cfme/web_ui/__init__.py

```diff
@@ -977,10 +977,11 @@ def table_in_object(table_title):
         table_title: Text in `p` element preceeding the table
     Returns: XPath locator for the desired table.
     """
+    # REVIEW: What uses this? I wasn't able to test it.
     # Description     paragraph with the text       following element    which is the table
     return (
         "//p[@class='legend' and normalize-space(text())='{}']"
-        "/following-sibling::*[1][@class='style3']").format(table_title)
+        "/following-sibling::table").format(table_title)
```

Comment:
Need some help finding one of this in the wild. I'm particularly concerned that `following-sibling[0]::*` might also be a table, though to comment above indicated that it's a paragraph, and this should work fine.